phpstan-baseline.neon

```diff
@@ -10,6 +10,16 @@ parameters:
 			count: 1
 			path: src/Persistence/Reflection/EnumReflectionProperty.php
 
+		-
+			message: "#^Method Doctrine\\\\Persistence\\\\Reflection\\\\EnumReflectionProperty\\:\\:getDeclaringClass\\(\\) return type with generic class ReflectionClass does not specify its types\\: T$#"
```

Comment:
This should be solved by making EnumReflectionProperty generic on the class name, as done in the stubs for ReflectionProperty in Psalm and phpstan instead of adding it in the baseline IMO.

Reply:
I agree, but I'm not sure how I can make the type flow from the constructor to there :thinking: I suppose I'm supposed to leverage `ReflectionProperty`'s template types, if there are some, but I don't know where to find their definitions.

Reply:
hmm, apparently, Psalm makes ReflectionProperty generic but phpstan does not. So I'm not sure solving this one is actually possible.